playbooks/main.yaml

```diff
@@ -0,0 +1,32 @@
+- hosts: all
```

Comment:
Is there any chance this file could go into `.zuul.d/`?

Reply:
Currently Zuul considers anything in ```.zuul.d``` configuration, so putting playbooks in here will result in configuration errors.

Two thoughts are that we could move it to a sibling hidden-directory, or we could enhance Zuul to obey a ```.zuul.ignore``` type file in directories so it knows they aren't config.  I'll try the first and investigate the second.

Reply:
`.zuul-playbooks/` is fine. just don't want to have a top level playbooks dir.

Reply:
> `.zuul-playbooks/` is fine. just don't want to have a top level playbooks dir.

Done like this for now